pkg/asset/ignition/machine/node.go

```diff
@@ -37,16 +37,19 @@ func pointerIgnitionConfig(installConfig *types.InstallConfig, rootCA []byte, ro
 			ignitionHost = net.JoinHostPort(installConfig.VSphere.APIVIP, "22623")
 		}
 	}
+	queryvals := url.Values{}
+	queryvals.Add("token", token)
```

Comment:
Move this inside the function where the URL is created.

Reply:
This is that function, right?

Reply:
Oh sorry misread.  Fixed.